### DISCUSSION TITLE
OpenTelemetry for Play Framework 1

### DISCUSSION THREAD

**Discussion by jvaldezwds** [playframework/play1#1461] [Ideas]
Hello

[Open Telemetry](https://opentelemetry.io/) is a library that is used to monitor applications. Basically there are two modes, [automatic](https://opentelemetry.io/docs/instrumentation/java/automatic/), which provides an already built in jar that can be attached to an application and start monitoring, and [manual](https://opentelemetry.io/docs/instrumentation/java/manual/), in which the source code is instrumented with code to provide the traces/metrics to monitor the application.

For the automatic mode, it supports Play Framework from 2.4, see [here ](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md)

There is a plan to add also this support to Play Framework 1, since it is a version that is still used and maintained?

Thanks

José Valdez

**Comment by ihostage** [comment]
I'm not sure that OT maintainers plan to add support of Play 1. But it's OpenSource project and I think they can consider pull request if someone will do that. Anyway, It's better to discuss that with them.

But I'm sure that maintainers of Play 2 don't plan to support OT for Play 1, as well as Play 1 itself, sorry 🤷‍♂️

**Reply by jvaldezwds** [reply]
Thanks for your reply!
